STUDIO/org.openl.rules.webstudio/resources/openl-default.properties

```diff
@@ -321,6 +321,11 @@ repo-default.production.local-repository-path = ${openl.home}/repositories/produ
 # Path to the repository folder where deployments are stored. The value must end with "/".
 repo-default.production.base.path=deploy/
 
+# Encryption algorithm for objects on the server side. Available algorithms: "AES256" and "aws:kms".
+# "AES256" is the base level of server side encryption configuration for S3 bucket at the rest using AES-256.
+# "aws:kms" next level of encryption, that provides additional level of encryption with AWS Key Management Service.
+repo-default.production.sse-algorithm=
+
```

Comment:
Move it to `org.openl.rules.repository.aws` Maven module

Reply:
done